src/playlist-handlers/youtube-playlist-handler.ts

```diff
@@ -0,0 +1,23 @@
+import { URLSearchParams } from "url";
+import ytpl from "ytpl";
+import { SongQueue } from "../song-queue/song-queue";
+
+export async function YoutubePlaylistToQueueItems(playlistUrl: string) {
+    const queue = SongQueue.get();
+    const urlParams = new URLSearchParams(playlistUrl);
+    const tracks = await (await ytpl (urlParams.get('list') || '')).items;
+    tracks.forEach(i => queue.addTrack(i.url, i.title));
+}
+
+export function IsYoutubePlaylist(playlistUrl: string) {
+    const playlistIndicators = ['/playlist', 'list='];
+    let is = false;
+    for (let indicator of playlistIndicators) {
+        if (playlistUrl.includes(indicator)) {
+            is = true;
+            break;
+        }
```

Comment:
`return playlistIndicators.some(indicator => playlistUrl.includes(indicator))`

Reply:
you right